### PR TITLE
Document how to licence work we produce

### PR DESCRIPTION
**Pull request by SteveMarshall**
When we build new things, we need to licence them appropriately so that others can use them safely. This is somewhat based on [GDS's styleguide](https://github.com/alphagov/styleguides/blob/master/licensing.md), but with a few MOJ-specific bits.

This also documents how to properly licence reused/incorporated code.

### CODE REVIEW

File: standards/licencing-software-or-code.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+Everything we produce or use, whether open or closed, should have an
+appropriate licence applied (as [indicated by the service
+manual][service manual licencing]). Without a licence, others (even
+within MOJ) have no way of knowing for sure if they can use it.
+
+At the MOJ, we use the [MIT License](https://opensource.org/licenses/MIT).
+
+[service manual licencing]: https://www.gov.uk/service-manual/technology/making-source-code-open-and-reusable#licensing-your-code
+
+## Applying the MIT License to our work
```
Comment (digitalronin) [inline standards/licencing-software-or-code.md]: Shouldn't this be;

## Applying the MIT Licence to our work

? The code uses 'licence' elsewhere.

File: standards/licencing-software-or-code.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+### Each repository should include a licence file
+
+This should be called `LICENCE` or `LICENCE.md`. "License" is the U.S.
+English spelling.
+
+GitHub will still show licence details for the British English spelling.
+
+Make sure the licence content is included in full, including the title
+"The MIT License", so that readers are quickly able to see what licence
```
Comment (digitalronin) [inline standards/licencing-software-or-code.md]: "The MIT Licence" ?

File: standards/licencing-software-or-code.md (hunk 1 above)
Comment (warmana) [reply standards/licencing-software-or-code.md]: It is correct to refer using License when talking about the MIT License specifically.

File: standards/licencing-software-or-code.md (hunk 2 above)
Comment (warmana) [reply standards/licencing-software-or-code.md]: As above.

File: standards/licencing-software-or-code.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
+---
+category: Building software
+expires: 2018-01-31
+---
+# Licensing software or code
+
+Everything we produce or use, whether open or closed, should have an
+appropriate licence applied (as [indicated by the service
+manual][service manual licencing]). Without a licence, others (even
+within MOJ) have no way of knowing for sure if they can use it.
```
Comment (warmana) [inline standards/licencing-software-or-code.md]: ... within the MOJ ...

Comment (warmana) [inline standards/licencing-software-or-code.md]: ... for sure if they are permitted by the originator to use the code or service, or under what terms or conditions.

File: standards/licencing-software-or-code.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+GitHub will still show licence details for the British English spelling.
+
+Make sure the licence content is included in full, including the title
+"The MIT License", so that readers are quickly able to see what licence
+is being used.
+
+### Use the correct copyright notice
+
+The Copyright is Crown Copyright; you can put "Ministry of Justice" in
```
Comment (warmana) [inline standards/licencing-software-or-code.md]: ... Copyright; optionally you can ...

File: standards/licencing-software-or-code.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+shown as a period from first to most recent update (e.g. 2015-2017).
+
+### Example
+
+`repo-audit` has a good [example of our licence](https://github.com/ministryofjustice/repo-audit/blob/master/LICENSE).
+
+## Reusing or incorporating others' work
+
+If a repository or package you want to reuse or build on doesn't have a
+licence, we can't be sure we legally have the right to use it.
```
Comment (warmana) [inline standards/licencing-software-or-code.md]: ... to use it, or to use it in the way we want.

File: standards/licencing-software-or-code.md (hunk 4 above)
Comment (SteveMarshall) [reply standards/licencing-software-or-code.md]: I think that's implied by “can”?

**Comment by SteveMarshall** [comment]
@warmana Made a couple of tweaks; mind re-reviewing?

File: standards/licencing-software-or-code.md (hunk 4 above)
Comment (warmana) [reply standards/licencing-software-or-code.md]: I'm being way too pedantic. Some purists say that 'can' refers to a technical ability to do something, not the permission to do so. As a stupid analogy, I 'can' put "Ministry of Sound" in the copyright notice, but it is neither correct nor optional to do so in this context.

**Pull request merged**